RELEASES.md

```diff
@@ -1,8 +1,22 @@
-# 0.4.0-beta1 (2023-05-10)
+# 0.4.0 LTS (2023-05-28)
 
-Changes compared to v0.3.5:
+No changes compared to v0.4.0-beta2.
 
-## Breaking Changes
+## Announcements
+
+The `v0.4.x` LTS series will be the last release to support older CMake and Rust versions.
+If necessary, fixes will be backported to the v0.4 branch. New features will not be
+actively backported after the v0.5 release, but community contributions are possible.
```

Comment:
idea: don't commit the next release v0.5
Maybe it would be time to move to 1.0 at some point?
But that's just details since we can still choose any version number regardless of what's written there.

Reply:
I'd say if you know that the next version will have breaking change, then continue with the 0.x series,  if not try to go with 1.x

We could have 1.0 LTS now, then 1.1  (non-lts)  that require new cmake and rust (because we could say that new version of cmake and rust are not breaking change)

Reply:
> I'd say if you know that the next version will have breaking change,

In that case I'll stick with 0.4 for now.